using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,331 @@
+
+The same argument must also be supplied to the [`add_drop_chunks_policy`
+function][api-add-drop-chunks] when creating a data retention policy for a
+hypertable with a continuous aggregate.  
+
+---
+
+### Best Practices [](best-practices)
+
+**Modifying the Materialization Hypertable**
+Advanced users may find the need to modify certain properties of the
+materialization hypertable (ie chunk size etc.) or to create further indexes,
+we can find the name of the materialization hypertable in the
+`timescaledb_information.continuous_aggregates` view ([API Docs][api-continuous-aggregates-info]).
+
+And then modify the materialization hypertable as if it were a normal
+hypertable. For instance, we may want to set the materialization hypertable's
+`chunk_time_interval` to something other than the default, this can be
+accomplished by running [`set_chunk_time_interval`][api-set-chunk-interval] on
+the materialization hypertable.  
+
+**Indexes on the Materialization Hypertable** 
+By default indexes on each group by column + the `time_bucket` column will be
+created  (i.e. in our example, we would create an index on {device_id, bucket}
+and if we had grouped by other columns, foo and bar, we would also create
+indexes on {foo, bucket} and {bar, bucket}). Setting
+`timescaledb.create_group_indexes` to `false` when creating the view will
+prevent this).  If we want to create additional indexes or drop some of the
+default ones, we can do so by creating or dropping the appropriate indexes on
+the materialization hypertable directly. 
+
+**Dealing with Timezones** 
+Functions that depend on a local timezone setting inside a continuous aggregate
+are not supported. We cannot cast to a local time because the timezone setting
+will change from user to user. So attempting to create a continuous aggregate
+like:
+```sql
+CREATE VIEW device_summary
+WITH (timescaledb.continuous)
+AS
+SELECT
+  time_bucket('1 hour', observation_time ) as bucket,
+  min(observation_time::timestamp) as min_time, --note the cast to localtime
+  device_id,
+  avg(metric) as metric_avg,
+  max(metric)-min(metric) as metric_spread
+FROM
+  device_readings
+GROUP BY bucket, device_id;
+``` 
+will fail. 
+
+Instead, we can use explicit timezones in our view definition like:
+```sql
+CREATE VIEW device_summary
+WITH (timescaledb.continuous)
+AS
+SELECT
+  time_bucket('1 hour', observation_time) as bucket,
+  min(observation_time AT TIME ZONE 'EST') as min_time, --note the explict timezone
+  device_id,
+  avg(metric) as metric_avg,
+  max(metric)-min(metric) as metric_spread
+FROM
+  device_readings
+GROUP BY bucket, device_id;
+DROP VIEW device_summary CASCADE;
+```
+Or we can cast to a timestamp on the way out of the view:
+```sql
+SELECT min_time::timestamp FROM device_summary;
+```
+---
+
+### Future Work [](future-work)
+We're releasing our first version of continuous aggregates in version 1.3, but
+it's a big feature and there's still lots more to do to make it all we know it
```

Comment:
Don't switch to much to active voice & pronouns. 

"The first version of continuous aggregations has been released in TimescaleDB v1.3., but we have a number of new capabilities and improvements already planned in the next releases.  Please find some of these forthcoming capabilities described below.  If you'd ...